src/store/index.js

```diff
@@ -1,24 +1,102 @@
 import Vue from "vue";
 import Vuex from "vuex";
 import movies from "../assets/movies.json";
-import { GETTERS } from "@/store/getters";
+import {
+  STATE,
+  GETTERS,
+  MUTATIONS,
+  ACTIONS,
+  EMPTY_STRING,
+  DEFAULT_SORT_OPTION,
+  DEFAULT_SEARCH_OPTION
+} from "@/store/storeConstants";
+import { TITLE_OPTION, GENRE_OPTION } from "@/core/constants";
+import { comparator } from "@/store/comparator";
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
```

Comment:
Since your state is getting a bit bloated, it might be worth looking into Vuex "modules".

Reply:
I'll do it in the next PR as a part of task-6